test/integration/integration_test.go

```diff
@@ -1364,3 +1366,56 @@ func installDebIfNotExists(r *require.Assertions, container, pkg string) {
 		),
 	))
 }
+
+func testBackupSpecifiedPartition(r *require.Assertions, ch *TestClickHouse) error {
+	log.Infof("testBackupSpecifiedPartition started")
+
+	testBackupName := fmt.Sprintf("test_backup_%d", rand.Int())
+	// Create table
+	ch.queryWithNoError(r, "CREATE TABLE default.t1 (dt DateTime, v UInt64) ENGINE=MergeTree() PARTITION BY toYYYYMMDD(dt) ORDER BY dt")
+	ch.queryWithNoError(r, "INSERT INTO default.t1 SELECT '2022-01-01 00:00:00', number FROM numbers(10)")
+	ch.queryWithNoError(r, "INSERT INTO default.t1 SELECT '2022-01-02 00:00:00', number FROM numbers(10)")
+	// Backup
+	r.NoError(dockerExec("clickhouse", "clickhouse-backup", "create_remote", "--tables=default.t1", "--partitions=20220101", testBackupName))
+
+	time.Sleep(10 * time.Second)
+
+	// TRUNCATE TABLE
+	ch.queryWithNoError(r, "TRUNCATE table default.t1")
+	r.NoError(dockerExec("clickhouse", "clickhouse-backup", "delete", "local", testBackupName))
+	r.NoError(dockerExec("clickhouse", "clickhouse-backup", "restore_remote", testBackupName))
+
+	time.Sleep(10 * time.Second)
```

Comment:
Remove this line, it has no sense
you return from `clickhouse-bakcup restore_remote` only after all restore complete